tests/prometheus_addon_test.go

```diff
@@ -19,14 +20,20 @@ func TestHelmPrometheusServerAddon(t *testing.T) {
 		manageNamespace: true,
 	}
 
+	// add Grafana namespace as it is expected by Prometheus stack
+	kubectlOptions := k8s.NewKubectlOptions("", "", addonData.namespaceName)
+	k8s.CreateNamespace(t, kubectlOptions, "grafana")
+	defer func() {
+		k8s.DeleteNamespace(t, kubectlOptions, "grafana")
+	}()
```

Comment:
Can we put every deletion down below on the test so it's easier to follow the logic? As you may have noticed we do not use `defer` exactly because of this reason. Thank you :)

Reply:
Done.